src/scss/includes/utils.scss

```diff
@@ -0,0 +1,3 @@
+.u-bold {
```

Comment:
Introduces a first generic utility class, to facilitate simple inline styling without having to target everything with a specific classname.